dashboard/src/pages/api/run.ts

```diff
@@ -108,8 +108,18 @@ async function runSql(sql: string, dbAlias: string, username: string, password:
       host: baseConnConfig.host,
       ssl: baseConnConfig.extra.ssl,
     });
+    // Based on https://node-postgres.com/apis/client#error
+    connTemp.on('error', e => {
+      logger.error?.('Connection error', {
+        app: 'run',
+        env: process.env.IASQL_ENV,
+        meta: {
+          error: e.message,
+          stack: e.stack,
+        },
+      });
+    });
     await connTemp.connect();
-    // await connTemp.query(dbMan.setPostgresRoleQuery(database));
```

Comment:
why is the commented out? Isn't just catching the error enough

Reply:
this line was commented, I just removed assuming is not necessary

Reply:
oh my bad, got it